apache/trafficcontrol
# Document undocumented Traffic Ops routes

## What does this PR (Pull Request) do?


This PR:

- [x] fixes #4290 by documenting the remaining routes listed
- [x] adds POST cdns/dnsseckeys/generate and GET, POST, PUT, and DELETE deliveryservice_requests to the Python client for Traffic Ops

## Which Traffic Control components are affected by this PR?


- Documentation
- Traffic Control Client - Python

## What is the best way to verify this PR?

Testing can be performed in 3 steps:
* Ensure that the docs build without errors or warnings
* Check whether the docs are accurate and easy to follow
* Test out the Python client for Traffic Ops to see if requests to the added routes work

## If this is a bug fix, what versions of Traffic Control are affected?

- master (df7a3a3d1c)
- 3.1.0
- 4.0.0 (RC4)

## The following criteria are ALL met by this PR



## Additional Information
